tutorials/download_mistral.md

```diff
@@ -28,3 +30,33 @@ pip install sentencepiece
 
 python chat/base.py --checkpoint_dir checkpoints/mistralai/Mistral-7B-Instruct-v0.1
 ```
+
+### Mixtral
+
+[Mixtral 8x7B](https://mistral.ai/news/mixtral-of-experts) is a pretrained generative Sparse Mixture of Experts model based on Mistral 7B.
+Mistral-8x7B outperforms Llama 2 70B on most benchmarks tested.
+
+Details about the data used to train the model or training procedure have not been made public.
+
+In order to use the Mixtral 7B model checkpoint, which requires about 94 GB of disk space, download the weights and convert the checkpoint to the lit-gpt format:
+
+```bash
+pip install huggingface_hub
+
+python scripts/download.py --repo_id mixtralai/Mixtral-8x7B-Instruct-v0.1
+
+python scripts/convert_hf_checkpoint.py --checkpoint_dir checkpoints/mistralai/Mixtral-8x7B-Instruct-v0.1
+```
+
+Due to the size of the model, currently only the [multi-device sequential generation](inference.md#run-a-large-model-on-multiple-smaller-devices) script can handle it.
+
+```bash
+pip install sentencepiece
+
+CUDA_VISIBLE_DEVICES=0,1,2,3 python generate/sequentially.py --checkpoint_dir checkpoints/mistralai/Mixtral-8x7B-Instruct-v0.1
```

Comment:
Why do we need to advise the user to set this env var? It doesn't fit well the Lightning style IMO. Can we just split the model up across all devices that Fabric receives as input, wdyt?

Reply:
Your suggestion is what's done currently.

However, depending on # cards, mem/card and model memory, you might want to limit the number of visible devices partition across less devices and be faster at the cost of higher memory usage per card.

Right now the sequential logic is dumb and doesn't maximize the memory used per card.

(note that this discussion should go in #815)

Reply:
Removed, I'll add this back in #815. Merging